### DISCUSSION TITLE
Things to describe in docs

### DISCUSSION THREAD

**Discussion by rafmudaf** [NatLabRockies/floris#826] [v4 Design Discussion]
Turbine operating models:
- "mixed"
  - What is the "mixed" operating mode?
  - How does it work?
  - What are the assumptions around it?
  - [floris_interface_integration_tests.test_disable_turbines](https://github.com/NREL/floris/blob/v4/tests/floris_interface_integration_test.py#L145) implies that you need the mixed operating model to disable the turbines, so this could use some clarity

**Comment by misi9170** [comment]
We can clean up the following and transfer to the docs when appropriate, just getting my thoughts down now while I'm here

### Mixed operation model
The `MixedOperationTurbine` model allows users to switch between various other operation models. Currently enabled are the `CosineLossTurbine` and `SimpleDeratingTurbine` operation models. For a given turbine `t` and given findex `i`, the `MixedOperationTurbine` allows users to evaluate _either_ the `CosineLossTurbine` _or_ the `SimpleDeratingTurbine`, switched by passing in _either_ a nonzero yaw misalignment _or_ a non-default power setpoint at that `(t, i)` pair. If _both_ a yaw misalignment and power setpoint are sent, a `ValueError` is raised. 

This also enables users to mix yaw misaligned turbines and disabling turbines, as disabling turbines is a special case of setting a power setpoint.

**Reply by misi9170** [reply]
In future, the `MixedOperationTurbine` model may contain other models to switch between on top of the `CosineLossTurbine` and `SimpleDeratingTurbine`; this functionality may be further enhanced in future releases. However, we do not intend to enable handling multiple different types of setpoints simultaneously (e.g. `yaw_angles` and `power_setpoints`) until we know how a turbine would respond to such a situation. We are interested in hearing about studies looking into this type of behavior.
